src/test/tcp/test_tcp.c

```diff
@@ -20,12 +20,19 @@
 #include <sys/uio.h>
 #include <time.h>
 #include <unistd.h>
+#include <glib.h>
+#include <sys/msg.h>
+#include <sys/types.h>
+#include "test/test_glib_helpers.h"
 
-#define USAGE "USAGE: 'shd-test-tcp iomode type'; iomode=('blocking'|'nonblocking-poll'|'nonblocking-epoll'|'nonblocking-select') type=('client' server_ip|'server')"
+#define USAGE "USAGE: 'shd-test-tcp iomode type [queuename]'; iomode=('blocking'|'nonblocking-poll'|'nonblocking-epoll'|'nonblocking-select') type=('client' server_ip|'server') [queuname=(default='/tcp_shadow_queue')]"
 #define MYLOG(...) _mylog(__FILE__, __LINE__, __FUNCTION__, __VA_ARGS__)
-#define SERVER_PORT 58333
 #define BUFFERSIZE 20000
 #define ARRAY_LENGTH(arr)  (sizeof (arr) / sizeof ((arr)[0]))
+// default queue filename for IPC communication
+#define QUEUENAME "./test-tcp"
```

Comment:
Unused?

Reply:
We need a filename with ftok

Reply:
I think we want to make it mandatory to get this from the command-line (see below), in which case we can drop this.